services/dumper.js

```diff
@@ -76,6 +76,13 @@ function Dumper(config) {
     return _.kebabCase(table);
   }
 
+  function getModelName(table) {
+    if (table === 'sessions') {
```

Comment:
[Notice] You should add a comment here to explain why thus edge case needs to be handled